proxies/live/apiproxy/targets/bref-target.xml

```diff
@@ -17,10 +17,7 @@
         <Name>OauthV2.VerifyAccessToken</Name>
       </Step>
       <Step>
-        <Name>Quota</Name>
-      </Step>
-      <Step>
-        <Name>SpikeArrest</Name>
+        <Name>FlowCallout.ApplyRateLimiting</Name>
```

Comment:
If remember well you can override the defaults limits adding a custom attribute to an app... did you try that? that should be quite straight forward and you can prove that this is actually working... (and write a test for it)

Reply:
I didn't do that and I don't know how